etc/encoding/opencast.properties

```diff
@@ -0,0 +1,243 @@
+####
+# Profile definitions for the encoding service.
+#
+# In order to understand how to configure a format, take a look a the
+# documentation below:
+#
+# profile.<format>.name:
+#     Name of the format, used in download dialogs of the frontend.
+#
+# profile.<format>.output:
+#     Type of the distribution format.
+#     [audio|visual|audiovisual|stream|enhancedaudio|image|imagesequence|cover|nothing]
+#
+# profile.<format>.suffix:
+#     Extension that will be appended to the download.
+#
+# profile.<format>.mimetype:
+#     Mime type used to provide proper content types.
+#
+# profile.<format>.input:
+#     Track types for which this format is applicable.
+#     [audio|visual|audiovisual|stream|enhancedaudio|image|imagesequence|cover|nothing]
+#
+# profile.<format>.jobload:
+#     The job load for this specific encoding profile.
+##
+
+# The video distribution profile, optimized for parallel encoding of multiple
+# qualities for adaptive streaming
+profile.adaptive-parallel.http.name = parallel encode 480p and 720p and conditionally 1080p and 2160p qualities for adaptive streaming
+profile.adaptive-parallel.http.input = visual
+profile.adaptive-parallel.http.output = visual
+profile.adaptive-parallel.http.jobload = 4
+profile.adaptive-parallel.http.suffix.480p-quality = -480p.mp4
+profile.adaptive-parallel.http.suffix.720p-quality = -720p.mp4
+profile.adaptive-parallel.http.suffix.1080p-quality = -1080p.mp4
+profile.adaptive-parallel.http.suffix.1440p-quality = -1440p.mp4
+profile.adaptive-parallel.http.suffix.2160p-quality = -2160p.mp4
+profile.adaptive-parallel.http.ffmpeg.command.if-height-geq-900 = \
+  -filter:v scale=w=1920:h='trunc(1920/dar/2)*2',setsar=1,fps=25 -pix_fmt yuv420p \
```

Comment:
While I'm otherwise absolutely in favor of using it, I'm wondering if `force_divisble_by` is still too new an option. If I install ffmpeg on my Fedora 32 via packet manager, the version I'm getting (4.2) does not yet include it.

Reply:
It's over a year old now, we control most FFmpeg builds on servers and can just require a specific version for a new major version.